DESCRIPTION

```diff
@@ -26,7 +26,8 @@ Imports:
     jsonlite,
     stats,
     git2r (>= 0.11.0),
-    withr
+    withr,
+    rhub
```

Comment:
Don't know how heavy-weight rhub is, but maybe this should be in `Suggests()` with a `devtools::check_suggested()`.